gems/smithy/lib/smithy/welds/endpoints.rb

```diff
@@ -8,7 +8,7 @@ def pre_process(model)
         id, service = model['shapes'].select { |_k, s| s['type'] == 'service' }.first
         return if service['traits'] && service['traits']['smithy.rules#endpointRuleSet']
 
-        puts "Adding default endpoint rules to #{id}"
+        say_status :insert, "Adding default endpoint rules to #{id}", :yellow
```

Comment:
Probably a broader discussion than this PR - but does thor give us the ability to support log levels? Rather than just an on/off we may want to be able to do like debug/info/error or something.

Reply:
No, it doesn't. I investigated having a Logger and a configured log level. We could do that instead - but create_file will use say_status, so we would have to quiet everything always, and do our own logging. I'm not sure if we care to add logging output since generations are quick one-time invokes every so often. Would this be loaded in an always-running process you think?

Reply:
Other than for testing - no I dont really expect usage in an always running process.  I think most usage would come through invoking the CLI or through our own release process.  

I'm just thinking through what our release would look like - I think we generally don't want to log the create for every file, that ends up being huge for all of AWS services, but we likely want to be able to log warnings of some sort (like unknown traits? invalid formatting in docs?  I'm not quite sure).  

For now - I think just having the quiet option is reasonable.

Reply:
I'll look into options in the future. There is also a `say_error` that we could conditionally use based on options. I could put a logger on plan.